core/services/relay/evm/method_binding.go

```diff
@@ -124,6 +124,8 @@ func (m *methodBinding) blockNumberFromConfidence(ctx context.Context, confidenc
 		return big.NewInt(finalized.Number), nil
 	} else if confirmations == evmtypes.Unconfirmed {
 		return big.NewInt(latest.BlockNumber()), nil
+	} else if confirmations == evmtypes.Latest {
```

Comment:
I think that you can change `Unconfirmed` to use nil since `Latest` and `Unconfirmed` are the same thing here. The only difference is that right now the `latest` tag is used in `eth_getBlockByNumber` rpc call by head tracker, instead of in `CallContract`.

Reply:
Thanks for the suggestion, that's the change I've made